buildguide_EN.md

```diff
@@ -0,0 +1,464 @@
+
+![](./images/parts11.jpg)
+
+### Key socket
+
+A socket that allows you to insert and remove key switches without soldering. The socket itself needs to be soldered.
+
+![](./images/parts12.jpg)
+
+### RP2040 Zero
+
+A microcomputer board equipped with a microcomputer called RP2040.
+
+![](./images/parts13.jpg)
+
+### Rotary encoder
+
+Can detect rotation and depression.
+Used for volume controls, mouse wheels, etc. 
+
+![](./images/parts14.jpg)
+
+### RGB LED (SK6812MINI-E)
+
+Full color RGB LED.
+Soldered to the board in inverted position.
+
+![](./images/parts15.jpg)
+
+### Rubber feet
+
+Stick them on the bottom of the case to prevent slipping.
+
+### USB-C cable
+
+For communication with a computer.
+Depending on the environment, it may be C to C or A to C.  
+
+### Flathead screw 2.1x10
+
+Used for both the final case and the _workbench_ mode
+
+### Case
+
+This is the case.
+
+### Rotary encoder knob
+
+This is the knob for the rotary encoder.
+
+![](./images/parts20.jpg)
+
+### Large rotary encoder knob
+
+This is another knob for the rotary encoder.
+Use this when you need to keep turning it with your fingers.
+
+![](./images/parts21.jpg)
+
+### Joystick knob
+
+This is a knob for the joystick.
+
+![](./images/parts22.jpg)
+
+
+# Soldering/assembly
+
+The following describes soldering and assembly.
+For each item, some work on the front side and some work on the back side.
+The badge below indicates the orientation at the start of work, so please refer to it. 
+
+* ![Work surface - front](https://img.shields.io/badge/Work_surface-front-2ea44f)
+* ![Work surface - back](https://img.shields.io/badge/Work_surface-back-a42e4f)
+
+
+## Assembling the work bench
+
+Assemble the work bench to make soldering smoother.
+If you have the person next to you hold the bottom plate, assembly will go more smoothly, so please work together.
+
+![](./images/bg01.jpg)
+
+Screw the `solder supports` to the bottom part of the case. Perform the action on a flat and stable surface like a table. This is the `workbench mode`.
+
+
+## ![Work surface - back](https://img.shields.io/badge/Work_surface-back-a42e4f) insert the diodes x 12
+
+Use the `diode bender` to bend the diodes and insert them into the back of PCB.
+
+![](./images/bg06.jpg)
+
+Please pay attention to the orientation of the diodes as they have polarity and will not work if inserted the wrong way.
+As shown in the photo on the right, match the direction of the silk-printed triangular mark with the direction of the black side of the diode (arrow). 
+
+![](./images/bg07.jpg)
+![](./images/bg08.jpg)
+
+Solder one side.
+If the position is significantly off, heat it again to melt the solder and adjust the position.
+
+![](./images/bg09.jpg)
+
+Complete once both sides are soldered.
+
+![](./images/bg10.jpg)
+
+#### Orientation
+The correct orientation of the LEDs is to match the notch on one of its legs to the corner of the solkscreen.
+
+![](./images/bg10_detail.jpg)
+
+
```

Comment:
This part should be moved to the SK6812MINI-E section.

Reply:
I moved it to the top, to the LED section, is that the one you were refering to?

Reply:
What I had in mind was the following location. However, the place you changed it to now is also fine.

https://github.com/sago35/tinygo_keeb_workshop_2024/blob/8f51e85f88fdf0024402dde3293f351f5e62baeb/buildguide_EN.md#-rgb-led-sk6812mini-e